dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/sms/config/SmsGateway.java

```diff
@@ -138,29 +138,22 @@ public HttpStatus send( String urlTemplate, HttpEntity<?> request, HttpMethod ht
 
     public OutboundMessageResponse wrapHttpStatus( HttpStatus httpStatus )
     {
-        GatewayResponse gatewayResponse;
+        GatewayResponse gatewayResponse = getGatewayResponse( httpStatus );
 
         OutboundMessageResponse status = new OutboundMessageResponse();
-
-        if ( OK_CODES.contains( httpStatus ) )
```

Comment:
Is this just a refactoring? To me it seems that the code already behaves in the same way, unless I miss a detail 😅